Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1203,15 +1203,27 @@ public long GetNextTxNonce(Address address)
                 block
             );
             IReadOnlyList<ActionEvaluation> evaluations = null;
+            DateTimeOffset evaluateActionStarted = DateTimeOffset.Now;
             evaluations = BlockEvaluator.EvaluateActions(
                 block,
                 stateCompleters ?? StateCompleterSet<T>.Recalculate
             );
+            _logger.Debug(
```

Comment:
Is it okay we left these logs as debug level (not verbose)?

Reply:
`Verbose` would be better as client code can filter the log level to record.

Reply:
You mean it will be better to log it as a verbose level for only the line not other lines in those changes. right?

Reply:
I believe new logs you added in this patch should be recorded as the `Verbose` level.